lib/cc/engine/analyzers/violation.rb

```diff
@@ -8,8 +8,8 @@ module Analyzers
       class Violation
         attr_reader :issue
 
-        def initialize(base_points, issue, hashes)
-          @base_points = base_points
+        def initialize(language, issue, hashes)
+          @language = language
```

Comment:
Ah, missed your second comment before. 

I'm super loathe to use `strategy` but will change it to `language_strategy` if you feel strongly. :)